docs/source/intervention_models/maternal_supplementation/index.rst

```diff
@@ -107,48 +111,62 @@ Given the low utilization of MMS and BEP relative to IFA, we assume that baselin
+
+    "Overall, for women receiving *any* intermittent iron regimen (with or without other vitamins and minerals) compared with a daily regimen there was no clear evidence of differences between groups for any infant primary outcomes: low birthweight (average risk ratio (RR) 0.82; 95% confidence interval (CI) 0.55 to 1.22; participants = 1898; studies = eight; low quality evidence), infant birthweight (mean difference (MD) 5.13 g; 95% CI ‐29.46 to 39.72; participants = 1939; studies = nine; low quality evidence), premature birth (average RR 1.03; 95% CI 0.76 to 1.39; participants = 1177; studies = five; low quality evidence), or neonatal death (average RR 0.49; 95% CI 0.04 to 5.42; participants = 795; studies = one; very low quality). None of the studies reported congenital anomalies." (Abstract)
+
+  Note that the :ref:`BEP simulation <2017_concept_model_vivarium_gates_bep>` used the proportion of women who took antenatal iron for 90+ days for baseline coverage.
+
+  The confidence interval was calculated using the formula :math:`1.96 \times \sqrt{\frac{p \times (1 - p)}{n}}`.
 
 Vivarium Modeling Strategy
 --------------------------
 
 The maternal supplementation intervention is administered to mothers and impacts both the mother and infant. To model the impact of the intervention on either child or maternal outcomes, simulant attributes for maternal nourishment exposure (BMI/x-factor) and maternal ANC attendance exposure are required. Additionally, to model the impact on child growth, child growth exposures are required. To model the impact on maternal mortality, a maternal hemoglobin exposure value is required. This intervention model requires the additional simulant attribute of maternal supplement regimen.
 
-For the implementation of the intervention in an alterative scenarios, we will model BEP supplementation among undernourished mothers and MMS supplementation among adequately nourished mothers rather than IFA supplementation alone, as demonstrated in the following decision tree.
-
-.. note::
+For the implementation of the intervention in an alterative scenarios, we will model BEP supplementation among undernourished mothers and MMS supplementation among adequately nourished mothers rather than IFA supplementation alone, as demonstrated in the following decision tree. 
 
-  This decision tree assumes a complete transition from IFA to targeted BEP/MMS. Alternative intervention implementations may be considered.
+The "undernourished" and "adequately nourished" decision node should be based on the x-factor exposure for the acute malnutrition simulation, :ref:`documented here <2019_concept_model_vivarium_ciff_sam>`; however, other risk exposure definitions such as pre-pregnancy body mass index of less than 18.5 should be considered for other models.
```

Comment:
This is a bit different that what I was thinking, since I was imagining X-Factor to be unobservable, and therefore not usable for risk targeting.  Let's discuss.